### PR TITLE
Add script to compress video and convert to gif file

### PR DESCRIPTION
**Pull request by bmarty**
Small script to compress videos.
Gif is not always smaller than the mp4, but we can try to iterate later.

### CODE REVIEW

File: tools/compressVideo.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+echo "Converting file $1"
+file=$(echo $1 | sed 's/\.[^.]*$//')
+ffmpeg -i $1 -filter_complex "[0:v] fps=12,scale=480:-1,split [a][b];[a] palettegen [p];[b][p] paletteuse" $file-tmp.gif
```
Comment (ouchadam) [inline tools/compressVideo.sh:22]: I ended up with two versions of this script, one with scale=480 and a small version with 240 for when the gif ends up bigger than 10mb :sob:

File: tools/compressVideo.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+echo "Converting to Gif"
+gifsicle -O3 --lossy=80 -o $file.gif $file-tmp.gif
```
Comment (ouchadam) [inline tools/compressVideo.sh:24]: do you think it's worth mentioning the dependencies on `ffmpeg` and `gifsicle`?

**Review by ouchadam — APPROVED** [review]

File: tools/compressVideo.sh (hunk 2 above)
Comment (bmarty) [reply tools/compressVideo.sh:24]: If missing users will find out fast enough (I had to install `gifsicle` on my side). This script is just for us. No doc, no comment, no robustness, nothing :). Ah yes, some echoing!

File: tools/compressVideo.sh (hunk 1 above)
Comment (bmarty) [reply tools/compressVideo.sh:22]: There is room for improvement, for instance test the output file size and start again with 240.

**Pull request merged**